src/modeler/AutoRest.Swagger/Validation/TrackedResourceValidation.cs

```diff
@@ -138,7 +138,7 @@ private bool HandleSchema(Schema schema, Dictionary<string, Schema> definitions)
             return true;
         }
 
-        private bool listByXCheck(List<Operation> getOperations, Regex regEx, string definitionKey, Dictionary<string, Schema> definitions)
+        private bool listByXCheck(IEnumerable<Operation> getOperations, Regex regEx, string definitionKey, Dictionary<string, Schema> definitions)
```

Comment:
don't know if I get you, how does changing the name of this *private* method cause changes "beyond the scope of this PR"?